src/mlpro/bf/examples/howto_bf_023_normalizers.py

```diff
@@ -0,0 +1,126 @@
+ ## -------------------------------------------------------------------------------------------------
+## -- Project : MLPro - A Synoptic Framework for Standardized Machine Learning Tasks
+## -- Package : mlpro.bf.examples
+## -- Module  : howto_bf_023_normalizers.py
+## -------------------------------------------------------------------------------------------------
+## -- History :
+## -- yyyy-mm-dd  Ver.      Auth.    Description
+## -- 2022-09-16  0.0.0     LSB      Creation
+## -- 2022-09-25  1.0.0     LSB      Release of first version
+## -------------------------------------------------------------------------------------------------
+
+"""
+Ver. 1.0.0 (2022-09-25)
+Example file for demonstrating the use of MLPro's normalizer for normalizing and de-normalizing data.
+"""
+
+
+from mlpro.bf.math.normalizers import *
+from mlpro.bf.various import Log
+
+# checking for internal unit tests
+if __name__ == '__main__':
+    p_logging = Log.C_LOG_ALL
+else:
+    p_logging = Log.C_LOG_NOTHING
+
+
+# Creating Numpy dummy Dataset
+my_dataset = np.array(([45,-7,65,-87],[21.3,47.1,-41.02,89],[0.12,98.11,11,-56.01]))
+
+
+# Creating a dummy set with dummy dimensions
+my_data = Set()
+my_data.add_dim(Dimension(p_name_short='1', p_boundaries=[10,19]))
+my_data.add_dim(Dimension(p_name_short='2', p_boundaries=[-9,10]))
+
+
+# Creating a dummy element to normalize
+my_state = Element(my_data)
+my_state.set_values([19,8])
+
+
+
+
+## -------------------------------------------------------------------------------------------------
+## -------------------------------------------------------------------------------------------------
+class MyNormalizerMinMax(NormalizerMinMax, Log):
+    def __init__(self):
```

Comment:
That's interesting: the constructor doesn't define a parameter p_logging. Obviously there is an implicite/inherited definition of it. However, I recommend to explicitely define it here. Otherwise it's hard to understand for others.